### PR TITLE
OSDOCS-3301: Fixing typos/nits

### PR DESCRIPTION
**Pull request by skrthomas**
Version(s):

4.11+
Issue:

This PR was Manushri's during her internship. I noticed it went stale but it had peer review done and lgtm in comments. However, there was one comment straggling by @xingxingxia that I missed, and the PR was merged. So this PR is to fix those nits that he pointed out in that comment. 
https://github.com/openshift/openshift-docs/pull/62980
Link to docs preview:

https://68015--docspreview.netlify.app/openshift-enterprise/latest/nodes/pods/nodes-pods-secrets#nodes-pods-secrets-creating-web-console-secrets_nodes-pods-secrets

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/nodes-pods-secrets-creating-web-console-secrets.adoc [hunk 1]
```diff
@@ -20,14 +20,14 @@ For example:
...
-    name: example
-    namespace: <namespace>
+  name: example
+  namespace: <namespace>
 type: Opaque <1>
 data:
-    username: <username>
-    password: <password>
+  username: <base64 encoded username>
+  password: <base64 encoded username>
```
Comment (xingxingxia) [inline modules/nodes-pods-secrets-creating-web-console-secrets.adoc:28]:
```suggestion
  password: <base64 encoded password>
```

File: modules/nodes-pods-secrets-creating-web-console-secrets.adoc [hunk 2]
```diff
@@ -20,14 +20,14 @@ For example:
...
 stringData: <2>
-    hostname: my app.mydomain.com 
+    hostname: myapp.mydomain.com
```
Comment (xingxingxia) [inline modules/nodes-pods-secrets-creating-web-console-secrets.adoc:30]:
```suggestion
  hostname: myapp.mydomain.com 
```

File: modules/nodes-pods-secrets-creating-web-console-secrets.adoc (hunk 1 above)
Comment (skrthomas) [reply modules/nodes-pods-secrets-creating-web-console-secrets.adoc]: Addressed

File: modules/nodes-pods-secrets-creating-web-console-secrets.adoc (hunk 2 above)
Comment (skrthomas) [reply modules/nodes-pods-secrets-creating-web-console-secrets.adoc]: Ah missed this one completely. TY!

**Comment by skrthomas** [comment]
@xingxingxia Addressed your comments. Sorry about those!

**Comment by xingxingxia** [comment]
/lgtm
Thank you!

**Pull request merged**